plugins/modules/notification/slack.py

```diff
@@ -310,7 +358,8 @@ def main():
             parse=dict(type='str', default=None, choices=['none', 'full']),
             validate_certs=dict(default=True, type='bool'),
             color=dict(type='str', default='normal'),
-            attachments=dict(type='list', required=False, default=None)
+            attachments=dict(type='list', required=False, default=None),
+            blocks=dict(type='list', required=False, default=None),
```

Comment:
```suggestion
            blocks=dict(type='list', elements='dict'),
```
`required=False` and `default=None` are the default values and are not needed.

Reply:
Should I clean up other params which have the same `required=False, default=None` pattern (`domain`, `msg`, and `attachments`), and add `elements` definitions for other "list" params?

Reply:
That would be great, but please do this in a follow-up PR. It's better to separate features from cleanup :)